dcpy/library/templates/dob_cofos.yml

```diff
@@ -4,6 +4,9 @@ dataset:
   source:
     script:
       path: .library/upload/dob_cofos.csv
+      ## This field must be updated manually when archiving. Should eventually be refactored out of here
+      ## Do not check in a specific version, leave blank version check in - this has led to issues in the past
+      previous_version: 20240105
```

Comment:
Seemed to make more sense to have this be an input to the script rather than tucked away in "info"